### PR TITLE
Documentation fixes

### PR DESCRIPTION
**Pull request by priyanshi-gupta**
Documentation fixes to ensure parity with other framework docs for public beta

### What and why?

What changes does this pull request introduce and why is it necessary?

### How?

If needed, a description of how this PR accomplishes what it does.

### Review checklist

- [ ] This pull request has appropriate unit and / or integration tests 
- [ ] This pull request references a Github or JIRA issue

### CODE REVIEW

**Review by fuzzybinary — DISMISSED** [review]

File: packages/datadog_flutter_plugin/README.md [hunk 1]
```diff
@@ -1,23 +1,12 @@
...
-
-| iOS SDK | Android SDK | Browser SDK |
-| :-----: | :---------: | :---------: |
-| 1.10.0 | 1.12.0-alpha2 | ❌ |
+## Overview
 
-[//]: # (End SDK Table)
+Datadog Real User Monitoring (RUM) enables you to visualize and analyze the real-time performance and user journeys of your Flutter application’s individual users.
 
+RUM supports monitoring for Flutter Android and iOS mobile applications.
```
Comment (alai97) [inline packages/datadog_flutter_plugin/README.md:9]:
```suggestion
RUM supports monitoring for mobile Flutter Android and iOS applications.
```

File: packages/datadog_flutter_plugin/README.md [hunk 2]
```diff
@@ -1,23 +1,12 @@
-## Overview
-
-Datadog Real User Monitoring (RUM) enables you to visualize and analyze the real-time performance and user journeys of your Flutter application’s individual users.
-
-## Platform Support
-
-| Android | iOS |  Web | MacOS | Linux | Windows |
-| :-----: | :-: | :---: | :-: | :---: | :----: |
-|   ✅    | ✅  |  🚧   | ❌  |  ❌   |   ❌   |
+<div class="alert alert-info"><p>Flutter monitoring is in beta</p>
```
Comment (alai97) [inline packages/datadog_flutter_plugin/README.md:1]:
```suggestion
<div class="alert alert-info"><p>Flutter monitoring is in beta.</p>
```

**Review by alai97 — DISMISSED** [review]
Looks good, two copy nits!

**Comment by fuzzybinary** [comment]
@priyanshi-gupta Just noticed these should probably target Master not Develop.

**Comment by fuzzybinary** [comment]
@priyanshi-gupta I'm going to close this and re-open with a cherry pick on "main".

**Pull request closed without merging**